R/stan_module.R

```diff
@@ -213,12 +187,13 @@ read_file <- function(filename) {
 }
 
 
+#' @importFrom assertthat assert_that
```

Comment:
@charksong  I think the issue I mentioned above about the NAMESPACE file is coming from this line. I think this needs to be placed below the function description